### DISCUSSION TITLE
'Empty Last query' problem

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#735] [General]
- you set the "Empty Last Query", but when you open query window, Short time showing last query and cleaning. 
- It's a very old problem that's been going on since wox.
- It's a subtly annoying problem, so have to fix it.
- https://github.com/Wox-launcher/Wox/issues/2795

**Hint**
- https://github.com/Wox-launcher/Wox/issues/2795#issuecomment-633171613
- Hotkeymapper has cleaning function.
- it doesn't cleaning when close window. It cleaning when new open.
- it have to clean it when close it.

**Comment by jjw24** [comment]
tried solution
https://github.com/Flow-Launcher/Flow.Launcher/pull/749
